[program:gunicorn_asgi]
;environment=PYTHONPATH=/home/postgres/.local/lib/python3.8/site-packages/,DJANGO_SETTINGS_MODULE=main.settings.docker
environment=PYTHONPATH=/home/postgres/.local/lib/python3.8/site-packages/
command=gunicorn -c ./gunicorn.py main.asgi:ws_application
directory=/server/websocket
numprocs=1           ; number of processes copies to start (def 1)
autostart = true     ; 在 supervisord 启动的时候也自动启动
startsecs = 1        ; 启动 1 秒后没有异常退出，就当作已经正常启动了
autorestart = true   ; 程序异常退出后自动重启
startretries = 3     ; 启动失败自动重试次数，默认是 3
user = root          ; 用哪个用户启动
stopsignal=INT               ; signal used to kill process (default TERM)
stopasgroup=true             ; send stop signal to the UNIX process group (default false)
killasgroup=true             ; SIGKILL the UNIX process group (def false)
;redirect_stderr = true          ; 把 stderr 重定向到 stdout，默认 false
;stdout_logfile_maxbytes = 20MB  ; stdout 日志文件大小，默认 50MB
;stdout_logfile_backups = 10     ; stdout 日志文件备份数
;stdout_logfile=/server/backend/django_server-main/logs/gunicorn.log       ; log 日志
;stderr_logfile=/server/backend/django_server-main/logs/gunicorn.error     ; 错误日志的绝对路径
